extension/modules/personas.js

```diff
@@ -134,7 +134,7 @@ var PersonaController = {
         }
     },
 
-    get_footer(document) {
+    get_footer: function(document) {
```

Comment:
This is actually added for SeaMonkey. It does not create any issue for Firefox. Without it SeaMonkey complains.